src/App.js

```diff
@@ -1,15 +1,17 @@
-import { Routes, Route, BrowserRouter as Router } from 'react-router-dom';
-import Home from './components/Home';
+import { Routes, Route } from 'react-router-dom';
+import NavBar from './components/NavBar';
+import Home from './components/Home/Home';
+import Details from './components/Details';
 
 function App() {
   return (
-    <Router>
-      <>
-        <Routes>
-          <Route path="/" element={<Home />} />
-        </Routes>
-      </>
-    </Router>
+    <>
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<Home />} />
```

Comment:
- [ ] While you have done a lot beautiful work on this project, let us not forget to keep to design guide, kindly make your project to look similar to the given design
your design :point_down: 
![Screenshot from 2023-11-18 19-27-06](https://github.com/Obimbo07/metrics-capstone/assets/66097371/6e67ceae-56e5-4414-844f-3d9bf004403b)
given design :point_down: 
![Screenshot from 2023-04-28 11-04-25](https://github.com/Obimbo07/metrics-capstone/assets/66097371/f69913b0-c98c-43e3-87cc-03adc8705b5d)